packages/checkout/shipping/shipping-rates-control/render-package-rate-option.tsx

```diff
@@ -3,33 +3,35 @@
  */
 import { decodeEntities } from '@wordpress/html-entities';
 import { DISPLAY_CART_PRICES_INCLUDING_TAX } from '@woocommerce/block-settings';
-import FormattedMonetaryAmount from '@woocommerce/base-components/formatted-monetary-amount';
 import { getCurrencyFromPriceResponse } from '@woocommerce/price-format';
+import FormattedMonetaryAmount from '@woocommerce/base-components/formatted-monetary-amount';
```

Comment:
This one gives me an error. `Casing of @woocommerce/base-components/formatted-monetary-amount does not match the underlying filesystem.`

Reply:
Does this error appear only in your editor and not in the CI? If you don't have other casing on this path `assets/js/base/components/formatted-monetary-amount/index.tsx` it might be a false positive. Found a [similar issue](https://github.com/benmosher/eslint-plugin-import/issues/720). Could you maybe try to update to eslint-plugin-import@v2.8.0 and see if it still happens.